### PR TITLE
desktop file

### PR DESCRIPTION
**Pull request by olanikegloria**
In this milestone I learnt to:
Understand how to create UIs adaptable to different screen sizes using media queries.
Understand how to parse a Figma design to create a UI.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -443,11 +475,13 @@ textarea::placeholder {
...
     margin-right: 100px;
     border-radius: 24px;
+    padding-left: 8px;
+    padding-right: 8px;
   }
 
   .ruby {
-    padding: 4px 20px;
-    margin-right: -1px;
+    /* padding: 4px 20px; */
```
Comment (Ibizugbe) [inline style.css]: Nice job so far👍. I suggest you take away every comment block of code in your project as it makes your code base look messy and less professional hence it is not a good practice👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @olanikegloria 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!👍

## Highlights✅✅✅

- [x] Linters are passing👍.
- [x] Porject matches the design👍.
- [x] Media queries have been used for the Desktop breakpoint in the CSS file👍.
- [x] Flexbox and Grid have ben used in order to layout the elements on the page.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] For future reference, I suggest you ensure your commit messages are descripriptive, meaningful and written in imperative tense😉. as this gives your project a more professional look👍. You can take a look at [this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to help you document your project professionally in future times👍.


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -432,8 +464,8 @@ textarea::placeholder {
   .container-1 img {
     width: 100%;
     height: 100%;
-    margin-top: 18px;
-    margin-left: 10px;
+    margin-top: 8px;
+    margin-bottom: -5px;
   }
 
   .container-1 {
```
Comment (AlphaNtihinduka) [inline style.css:471]: - [ ] [OPTIONA] Kindly ensure that the image is responsive to all the desktop screen sizes. I have noticed that the image at desktop screen size `774px` to about `930px` is squeezed due to the fixed width. I kindly suggest to consider using responsive width to make sure that you stick to the Figma template design 😎 
- [ ] [OPTIONAL] I also suggest that the middle list item of `ul` should be responsive and matches the Figma design.
- [ ] [OPTIONAL] On the second card where the image is aligned to the right, I have noticed that the language items are lapping over each other. Please try your best to ensure that they match the Figma template design
Your design 😎 
image

Figma design 😎 
image

File: style.css [hunk 3]
```diff
@@ -490,4 +523,47 @@ textarea::placeholder {
...
+  .section-4 {
+    background-color: #6070ff;
+    background-image: url("images/contact\ form\ background\ shapes@2x.png");
+    background-size: 90%;
+    background-position: center;
+    background-repeat: no-repeat;
+    border-top-left-radius: 100px;
+    width: 100%;
+    height: 100%;
+  }
```
Comment (AlphaNtihinduka) [inline style.css:536]: - [ ] As stated above about the responsiveness at all the desktop screen sizes. Please ensure that the Contact section is also responsive and its contents are user friendly and professional according to the Figma design ☺️.
Your design 😎 
image

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @olanikegloria 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**